less/index.less

```diff
@@ -57,3 +57,247 @@ html, body {
     height: 100%;
 }
 
+.button {
+	width: 18%;
+			margin: 1.8%;
+			padding: 1%;
+			border-radius: 3px;
+			background-color: @dark-background;
+
+			p {
+				color: @light-background;
+				font-family: 'Open Sans', sans-serif;
+				font-weight: 500;
+			}
+}
+
+@charcoal: #36454F;
+@light-background: #E8EBEE;
+@dark-background: #7F7F7F;
+@max800: ~"(max-width: 800px)"; //tablet
+@max500: ~"(max-width: 500px, min-width: 300px)"; // phone
+
+p font {
+	font-size: 62.5%;
+	font-family: 'Roboto', sans-serif;
+	color: @charcoal;
+}
+
+h1, h2, h3 {
+	font-family: 'Open Sans', sans-serif;
+	font-weight: 600;
+}
+
+.container {
+	max-width: 800px;
+	width: 100%;
+	margin: 0 auto;
+
+	@media @max800 {
+		width: 96%;
+		font-size: 2vw;			
+	}
+
+	nav {
+		width: 100%;
+		display: flex;
+		justify-content: flex-end;
+
+		@media @max800 {
+			width: 96%;			
+		}
+
+		.nav-items {
+			width: 35%;
+			padding: 2% 0 2% 2%;
+			display: flex;
```

Comment:
A mixin for flexbox specifications is common and could save you some space throughout this project :)